### ISSUE TITLE
Feature request:

### ISSUE THREAD

**Issue by b-d-m-p** [melink14/rikaikun#118]
Hi, 

Thank you for the great extension. 

I would like a keyboard shortcut to enable Rikaikun and a right click menu enable option. Would this be possible?

Cheers

**Comment by versusvoid** [comment]
Chrome by itself [provides](http://www.howtogeek.com/127162/how-to-create-custom-keyboard-shortcuts-for-browser-actions-and-extensions-in-google-chrome/) shortcuts for extensions.
Clobbering global context menu doesn't sound like good idea.

**Comment by melink14** [comment]
I agree that using native keyboard shortcuts is probably better for this.

**Comment by melink14** [comment]
Though one thing we could do is make a default shortcut and advertise it.
